lib/src/base-styles/_nx-tables.scss

```diff
@@ -306,10 +317,29 @@
   }
 }
 
-.nx-cell__sort-btn, .nx-cell__chevron-btn {
+// make button take up entire cell including its padding box
+@mixin btn-cell-sizing($left-padding, $right-padding) {
+  margin-left: calc(0px - #{$left-padding});
+  margin-right: calc(0px - #{$right-padding});
+  padding-left: $left-padding;
+  padding-right: $right-padding;
+}
+
+.nx-cell__sort-btn, .nx-cell__row-btn {
+  width: min-content;
+
```

Comment:
Should we add a `cursor: pointer;` to this class? Right now, hovering over both the sort cell and the row button icons don't show a pointer, making it seem like both the sort cell and the buttons are not clickable.

https://user-images.githubusercontent.com/7110862/134003686-ed73df59-52a5-482a-bb48-3e8b0e800434.mov

https://user-images.githubusercontent.com/7110862/134003713-25930c2c-b3b6-40ae-ae22-5817907ba2f0.mov

Reply:
> Should we add a cursor: pointer; to this class? Right now, hovering over both the sort cell and the row button icons don't show a pointer, making it seem like both the sort cell and the buttons are not clickable.

Good catch. baa93c79